scheduler/docs/configuration.asc

```diff
@@ -322,6 +323,11 @@ The configuration values are defined as follows:
   The maximum length for the unencoded string messages sent from a task via the Mesos executor HTTP API.
   The default is 512.
 
+`:portion`::
```

Comment:
This is nice but unnecessary.  We don't need docs because this isn't a permanent feature.  If it is going to be permanent, the user should be able to define what executor makes up the other portion and all portions should add up to 1.